### ISSUE TITLE
Braille: Avoid Repeating Nemeth Switch Indicators

### ISSUE THREAD

**Issue by mcantino1** [PreTeXtBook/pretext#1903]
We've discussed this at length, but I never submitted an actual issue for this.

Braille translation programs tend to wrap all bits of math in individual Nemeth Switch Indicators, but transcribers are taught to switch codes as infrequently as possible, while maintaining clarity. 

If a transcriber encounters several bits of math strung together, they will switch into Nemeth, transcribe all of the math content, and then switch out of Nemeth once the "literary" content resumes. Translation software will typically enclose each individual expression in switch indicators, which introduces a lot of clutter into a transcription. 

Basically, we should try to recognize strings of math content, and then try to enclose all of that content in a single set of switch indicators.

**Comment by rbeezer** [comment]
Perhaps easier than #1904, or they should be done simultaneously.
